nemo/collections/asr/data/audio_to_text_lhotse_prompted.py

```diff
@@ -32,6 +32,7 @@ class PromptedAudioToTextMiniBatch:
     prompt_lens: torch.Tensor
     prompted_transcript: torch.Tensor
     prompted_transcript_lens: torch.Tensor
+    cuts: CutSet
```

Comment:
```suggestion
    cuts: CutSet | None = None
```

let's make this optional